tests/integration/test_rest.py

```diff
@@ -143,13 +147,20 @@ async def test_avro_publish_primitive_schema(rest_async_client, admin_client):
 
 async def test_avro_publish(rest_async_client, registry_async_client, admin_client):
     tn = new_topic(admin_client)
-    other_tn = new_topic(admin_client)
-    await wait_for_topics(rest_async_client, topic_names=[tn, other_tn], timeout=NEW_TOPIC_TIMEOUT, sleep=1)
+    await wait_for_topics(rest_async_client, topic_names=[tn], timeout=NEW_TOPIC_TIMEOUT, sleep=1)
     header = REST_HEADERS["avro"]
     # check succeeds with 1 record and brand new schema
-    res = await registry_async_client.post(f"subjects/{other_tn}/versions", json={"schema": second_schema_json})
-    assert res.ok
-    new_schema_id = res.json()["id"]
```

Comment:
Is this is not covered anymore, or the behaviour tested previously has been replaced?

Reply:
Now it is, thanks!